plug.vim

```diff
@@ -986,6 +986,10 @@ function! s:update_impl(pull, force, args) abort
   let s:clone_opt = get(g:, 'plug_shallow', 1) ?
         \ '--depth 1' . (s:git_version_requirement(1, 7, 10) ? ' --no-single-branch' : '') : ''
 
+  if has('win32unix')
+    let s:clone_opt .= ' -c core.eol=lf -c core.autocrlf=input'
```

Comment:
I see, thanks for the clarification. I'll just go ahead and merge the patch. And yes, it would be nice if you could post a link to the conversation for future reference.